packages/tables/frontend/sirius-components-tables/src/table/row/ResizeRowHandler.tsx

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2024 CEA LIST.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { memo, useRef } from 'react';
+import { makeStyles } from 'tss-react/mui';
+import { useTableMutations } from '../../graphql/mutation/useTableMutation';
+import { ResizeRowHandlerProps } from './ResizeRowHandler.types';
+
+const useStyles = makeStyles()((theme) => ({
+  rowMain: {
+    '&:hover': {
+      backgroundColor: theme.palette.action.hover,
+    },
+  },
+  cell: {
+    '&:hover': {
+      outline: `1px solid ${theme.palette.action.selected}`,
+    },
+  },
+  handler: {
+    position: 'absolute',
+    margin: 0,
+    backgroundColor: '#B3BFC5',
+    borderColor: '#B3BFC5',
+    borderRadius: '2px',
+    width: '24px',
+    height: '4px',
+    bottom: 0,
+    left: '15px',
+    cursor: 'row-resize',
+  },
+}));
+
+interface DragState {
+  isDragging: boolean;
```

Comment:
Move this interface in `ResizeRowHandler.types.ts`

Reply:
Done